examples/test_BA.ipynb

```diff
@@ -87,7 +87,8 @@
     "        cam = rec.cameras[im.camera_id]\n",
     "        for p in im.points2D:\n",
     "            cost = pyceres.factors.BundleAdjustmentCost(cam.model_id, p.xy, im.qvec, im.tvec)\n",
-    "            prob.add_residual_block(cost, loss, rec.points3D[p.point3D_id].xyz, cam.params)\n",
+    "            camp = cam.params.astype(np.uint8)\n",
```

Comment:
Why the casting here?

Reply:
Ah crap that is a test artifact. We can remove this line.